app/utils/application-state-utils.ts

```diff
@@ -268,7 +280,17 @@ interface DoThisNextSection {
  * @param req the request containing the session and appeal status
  */
 function getAppealStatus(req: Request) {
-  return (req.session.appeal.application.isAppealLate && req.session.appeal.appealStatus === 'appealSubmitted') ? 'lateAppealSubmitted' : req.session.appeal.appealStatus;
+  if (req.session.appeal.application.isAppealLate) {
+    if (req.session.appeal.appealStatus === 'appealSubmitted') {
+      return 'lateAppealSubmitted';
+    }
+    if (req.session.appeal.outOfTimeDecisionType) {
+      return 'lateAppealEnded';
```

Comment:
to be more precise shouldn't it be 'lateAppealRejected'?

Reply:
Agree, updated